code/GridFieldConfig_LimitedRelationEditor.php

```diff
@@ -1,12 +1,25 @@
 <?php
+
+namespace MBIE\GridFieldLimitRelationEditor;
```

Comment:
The SS4 docs suggest a namespace root of vendor\package so I'd suggest we change the namespace on this file to something like PatrickNelson\GridFieldLimitItems'.

Reply:
Done!